oca_dependencies.txt

```diff
@@ -1,4 +1,5 @@
 # List the OCA project dependencies, one per line
 # Add a repository url and branch if you need a forked version
+server-tools
```

Comment:
Runbot says:

```
except_orm: ('Error', u"You try to install module 'event_multi_image' that depends on module 'base_multi_image'.\nBut the latter module is not available in your system.")
```

That module is in `server-tools` now. Is it not enough to add it here?